install/mysql/glpi-empty.sql

```diff
@@ -5824,6 +5824,8 @@ CREATE TABLE `glpi_profiles` (
   `is_default` tinyint NOT NULL DEFAULT '0',
   `helpdesk_hardware` int NOT NULL DEFAULT '0',
   `helpdesk_item_type` text,
+  `use_mentions` tinyint NOT NULL DEFAULT '1',
+  `user_mentions_list` varchar(255) NOT NULL DEFAULT 'full',
```

Comment:
I think that a single INT field should be enough, and we need to use 3 constants corresponding to disabled, restricted and full.